Library/Homebrew/shims/linux/super/cc

```diff
@@ -189,7 +189,8 @@ class Cmd
     when /^-R(.+)?/
       # -R is not the correct way to pass rpaths to the linker
       val  = chuzzle($1) || enum.next
-      wl = "-Wl," unless mode == :ld
+      path = canonical_path(val)
+      wl   = "-Wl," unless mode == :ld
```

Comment:
What's the motivation for using `canonical_path` here? Here's my previous comment on `canonical_path`, which still stands:
https://github.com/Linuxbrew/brew/pull/328#issuecomment-291979672

The error referenced in this issue is
```
shims/linux/super/cc:221:in `keep_orig?': undefined method `start_with?' for nil:NilClass (NoMethodError)
```
https://gist.github.com/ozcan/b5b8f3dd8e9b5dff0c2a066c5a62ef60#file-03-pip-L1505